department-of-veterans-affairs/va.gov-cms
# VACMS-7877 cms announcement block

## Description

Closes #7877. 

## Testing done
- locally removed block from block layout placement, re-placed block with same spot/restrictions.
- exported config with updated block uuids
- log out, see that block is present locally.
- push to github... 
- see the block present on tugboat when logging in to claro theme, no remove/replace shenanigans needed.

This is essentially the workflow described in "option 1" here: https://www.drupal.org/project/drupal/issues/2756331#comment-11921622

## Screenshots
<img width="697" alt="Screen Shot 2022-02-17 at 12 36 40 PM" src="https://user-images.githubusercontent.com/11279744/154566589-d4fff0e2-c392-4192-b06f-6dcd228b03c1.png">


## QA steps

As any user, visit the PR environment and see that the cms announcement block is present on the login page without needing to re-place the block in the block layout page.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
